core/include/scipp/core/variable.tcc

```diff
@@ -751,9 +751,10 @@ VariableView<underlying_type_t<T>> VariableProxy::castVariances() const {
   return requireT<DataModel<Vector<TT>>>(data()).variancesView(dims());
 }
 /**
-  Support explicit instantionatons for templates for generic Variable
+  Support explicit instantiations for templates for generic Variable and
+  VariableConstProxy
 */
-#define INSTANTIATE_VARIABLE(...)                                              \
+#define INSTANTIATE_VARIABLE_AND_SLICE(...)                                    \
```

Comment:
I'd keep the name `INSTANTIATE_VARIABLE`. This is in principle user-facing and should not be loaded with technical details.

Reply:
That's what I had originally for the same reasons you said. I'll change it back.